OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -121,7 +121,7 @@ private void init() {
         }
 
         // Add Firebase as an analytics provider if enabled in the config
-        if (config.isFirebaseEnabled())  {
+        if (config.getFirebaseConfig().isEnabled())  {
             analyticsRegistry.addAnalyticsProvider(injector.getInstance(FirebaseAnalytics.class));
```

Comment:
`ANALYTICS_ENABLED` key inside `FIREBASE` should be queried before adding this provider.